cfgov/v1/tests/test_blocks.py

```diff
@@ -105,11 +105,20 @@ def test_multiple_inputs_raises_valueerror(self):
             PlaceholderCharBlock.replace_placeholder(html, "a")
 
 
-class RAFToolBlockTestCase(TestCase):
+class RAFTBlockTestCase(TestCase):
     def test_render_no_placeholder_provided(self):
-        erap_tool_block = RAFToolBlock()
-        html = erap_tool_block.render(None)
+        raft = RAFTBlock()
+        html = raft.render(None)
         self.assertInHTML(
             '<div id="rental-assistance-finder" data-language="en"></div>',
             html,
         )
+
+    def test_county_threshold(self):
+        raft = RAFTBlock()
+        html = raft.render({"county_threshold": 20})
+        self.assertInHTML(
+            '<div id="rental-assistance-finder" data-language="en"'
+            " data-county-threshold=20></div>",
```

Comment:
Not a blocker to this PR, and thank you for adding this test, but: I can see that the optional `county_threshold` field isn't currently configured on [this page](https://www.consumerfinance.gov/coronavirus/mortgage-and-housing-assistance/renter-protections/find-help-with-rent-and-utilities/), so this HTML isn't used, but is this actually valid without quotes around the `20`? (I.e., shouldn't it be `data-county-threshold="20"` not `data-county-threshold=20`?) I notice that in Chrome the quotes get added, so maybe this doesn't matter.

Reply:
Yeah, `data-*` attributes will all get cast to strings when this html is ingested by a browser. For this test, it works because the jinja renderer just dumps the value you provide into the data-county-threshold attribute, but this would work in the browser as well because it would be rendered into the DOM as `"20"` (as you observed).